src/lib.rs

```diff
@@ -1764,7 +1905,7 @@ pub mod contract {
             let crosses_after_quote =
                 ((pool_after_quote.current_tick_index - pool_before.current_tick_index) / 10).abs();
             assert_eq!(crosses_after_quote, 0);
-            assert_eq!(quote_result.3.len() - 1, 19);
+            assert_eq!(quote_result.3.len() - 1, 146);
```

Comment:
quote_result should be definitely struct instead of tuple. It's hard to read what means `.3` , `.2`